### PR TITLE
[OneExplorer] Introduce hide command and button

### PR DESCRIPTION
**Pull request by dayo09**
This commit introduce hide command and button.
The nodes with canHide set as true are going to be hiden and unhiden by the toggle button

ONE-vscode-DCO-1.0-Signed-off-by: dayo09 <dayoung.lee@samsung.com>

-----

For #1061

### CODE REVIEW

**Comment by dayo09** [comment]
### GIF
0726-hide-new

**Review by yunjayh — APPROVED** [review]
LGTM

File: package.json [hunk 1]
```diff
@@ -96,6 +96,18 @@
...
       },
+      {
+        "command": "one.explorer.hideExtra",
+        "title": "Hide Extra Files",
+        "category": "ONE",
+        "icon": "$(eye)"
+      },
+      {
+        "command": "one.explorer.unhideExtra",
+        "title": "Unhide Extra Files",
```
Comment (jyoungyun) [inline package.json:107]: (optional) Unhide -> Show ?

**Review by jyoungyun — DISMISSED** [review]
LGTM

**Comment by llFreetimell** [comment]
Comments and conflicts..!

**Comment by dayo09** [comment]
@jyoungyun PTAL :-D

**Review by llFreetimell — APPROVED** [review]

**Review by hyunsik-yoon — APPROVED** [review]

**Comment by dayo09** [comment]
@llFreetimell Thanks :-D

**Pull request merged**